src/libsyntax/diagnostic_list.rs

```diff
@@ -93,6 +93,36 @@ For more information about the cfg attribute, read:
 https://doc.rust-lang.org/reference.html#conditional-compilation
 "##,
 
+E0538: r##"
+Attribute contains multiple of the same meta item.
```

Comment:
What about "Attribute contains same meta item more than once"? (it's a real question, I'm not an english native so it's just a suggestion!)

Reply:
That is slightly better wording.